pkg/controller/argocd/argocd_controller.go

```diff
@@ -178,26 +132,29 @@ func (r *ReconcileArgoCD) Reconcile(request reconcile.Request) (reconcile.Result
 		}
 		return reconcile.Result{}, err
 	}
-
 	reqLogger.Info("Route found for argocd-server", "Route.Host", argoCDRoute.Spec.Host)
 
-	consoleLink := newConsoleLink("https://"+argoCDRoute.Spec.Host, "ArgoCD")
+	argocCDRouteURL := fmt.Sprintf("https://%s", argoCDRoute.Spec.Host)
```

Comment:
There's probably nothing we can do here, but it feels weird for us to be calculating the route URL like this.

Reply:
Yes. I wonder why the protocol is not added in the route object. They externally append while displaying the link on the UI